### PR TITLE
alice: init at 2.003

### PR DESCRIPTION
**Pull request by ncfavier**
Open-source font by Ksenia Erulevich https://github.com/cyrealtype/Alice

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/alice/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation (attrs: {
+  pname = "Alice";
+  version = "2.003";
+
+  outputs = [ "out" "woff2" ];
+
+  src = fetchzip {
```
Comment (AndersonTorres) [inline pkgs/data/fonts/alice/default.nix:9]: `fetchFromGitHub` doesn't work?

Comment (ncfavier) [reply pkgs/data/fonts/alice/default.nix:9]: We could fetch the `v2.003` tag, but the source is slightly larger (6 MiB) than the release tarball (288 KiB) so I went with the latter.

File: pkgs/data/fonts/alice/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    url = with attrs; "https://github.com/cyrealtype/${pname}/releases/download/v${version}/${pname}-v${version}.zip";
```
Comment (AndersonTorres) [inline pkgs/data/fonts/alice/default.nix:10]: Well, I do not like to use `with`, so I suggest this below:

```suggestion
    url = let 
      inherit (attrs) pname version;
    in "https://github.com/cyrealtype/${pname}/releases/download/v${version}/${pname}-v${version}.zip";
```

Or maybe:

```suggestion
    url = with attrs;
       "https://github.com/cyrealtype/${pname}/releases/download/v${version}/${pname}-v${version}.zip";
```

File: pkgs/data/fonts/alice/default.nix [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation (attrs: {
+  pname = "Alice";
+  version = "2.003";
+
+  outputs = [ "out" "woff2" ];
```
Comment (AndersonTorres) [inline pkgs/data/fonts/alice/default.nix:7]: Is there any reason to this multiple output here?

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/data/fonts/alice/default.nix (hunk 3 above)
Comment (ncfavier) [reply pkgs/data/fonts/alice/default.nix:7]: See the discussion in https://github.com/NixOS/nixpkgs/issues/182465#issuecomment-1207234828: most desktop users don't need the webfonts.

File: pkgs/data/fonts/alice/default.nix (hunk 2 above)
Comment (ncfavier) [reply pkgs/data/fonts/alice/default.nix:10]: I think such a local use of `with` is fine.

**Pull request merged**